packages/nextjs/components/scaffold-eth/Contract/InputUI.tsx

```diff
@@ -37,7 +37,7 @@ const InputUI = ({ setForm, form, stateObjectKey, paramType }: TInputUIProps) =>
   }
 
   return (
-    <div className="flex space-x-2 items-end">
+    <div className="flex items-end border-2 border-gray-100 bg-base-200 rounded-xl text-primary/90 justify-between pr-3">
```

Comment:
I feel that the placeholder color it's too strong: we can make the color (or opacity) lighter... or use another color, like the grey in the Address component?

![image](https://user-images.githubusercontent.com/2486142/209861314-c75778af-664b-4831-b609-083b227a68b0.png)

(Not super important because a lot of this tweaks will come later on... but maybe we make it look better for now)